test/pageobjects/home.page.js

```diff
@@ -2,13 +2,37 @@ import Page from './page.js';
 
 class HomePage extends Page{
 
+    //define the elements
+
     get shopButton(){
         return $('#slide-6-layer-9');
     }
 
-    async clickShopButton() {
-        await this.shopButton.click();
+    get firstProduct(){
```

Comment:
Let's get a bit creative here. Let's figure out a way to select a product by providing a name or something alike.

Reply:
do you mean for selector that i used for shopButton or name of the method firstProduct()?

Reply:
I mean for both. Let's try using something like `productByName(productName)`.